client/apps/edit/components/admin/index.coffee

```diff
@@ -47,20 +48,24 @@ module.exports = React.createClass
 
   render: ->
     div { className: 'edit-admin' },
-
       section {
         className: 'edit-admin--verticals-tags' + @getActiveSection 'verticals-tags'
       },
         printTitle {
-          section: 'Verticals & Tagging'
+          section: if sd.CURRENT_CHANNEL.type is 'editorial' then 'Verticals & Tagging' else 'Tagging'
```

Comment:
We should use [this model function](https://github.com/artsy/positron/blob/master/client/models/channel.coffee#L15) to check if it is editorial. If we ever need to make a global change related to this, it's easier to handle the business logic through the model instead of looking for instances of `is 'editorial'`